swatchdog/README.md

```diff
@@ -0,0 +1,39 @@
+# Swatchdog
+
+## Get Started
+
+* Install poetry
+  ```shell
+  dnf install poetry
+  ```
+* Create and activate the virtual env
+  ```shell
+  poetry shell
+  ```
+* Install dependencies
+  ```shell
+  poetry install
+  ```
+* Run swatchdog
+  ```shell
+  poetry run swatchdog --version
```

Comment:
```suggestion
  poetry run swatchdog ee deploy
```

The --version flag is to display the swatchdog version. I think it's useful to explicitly add the "poetry run swatchdog ee deploy" command here (or add a full example section instead).